train.yaml

```diff
@@ -51,12 +51,12 @@ dataloader_options:
 
 # Added noise and reverb come from OpenRIR dataset, automatically
 # downloaded and prepared with this Environmental Corruption class.
-env_corrupt: !new:speechbrain.lobes.augment.EnvCorrupt
-    openrir_folder: !ref <data_folder>
-    reverb_prob: 0.3
-    noise_prob: 1.0
-    noise_snr_low: 10
-    noise_snr_high: 30
+# env_corrupt: !new:speechbrain.lobes.augment.EnvCorrupt
+#     openrir_folder: !ref <data_folder>
+#     reverb_prob: 0.3
+#     noise_prob: 1.0
+#     noise_snr_low: 10
+#     noise_snr_high: 30
```

Comment:
Since this is commented out only for specific experiments, I would suggest to uncomment these and you can comment them as per your experiment.